build/CopyFilesToStagingDir.ps1

```diff
@@ -110,6 +110,7 @@ PublishFile $FullBuildOutput\WindowsAppRuntime_DLL\Microsoft.Windows.System.Powe
 # Native (not managed, AppLocal / no MSIX)
 PublishFile $FullBuildOutput\WindowsAppRuntime_BootstrapDLL\Microsoft.WindowsAppRuntime.Bootstrap.dll $NugetDir\runtimes\lib\native\$Platform
 PublishFile $FullBuildOutput\WindowsAppRuntime_BootstrapDLL\Microsoft.WindowsAppRuntime.Bootstrap.pdb $NugetDir\runtimes\lib\native\$Platform
+PublishFile $FullBuildOutput\Microsoft.WindowsAppRuntime.Bootstrap.Net\Microsoft.WindowsAppRuntime.Bootstrap.Net.dll $NugetDir\runtimes\lib\native\$Platform
```

Comment:
>native [](http://example.com/codeflow?start=140&length=6)

why .net dll in native folder?

Reply:
It put it next to the current Bootstrap. dll for now. I suspect that's the wrong place, but not sure of the right one - yet. Easy enough to move once I do some digging, but I didn't want to block others (docs etc) while I do